packages/app/src/HDXMultiSeriesTimeChart.tsx

```diff
@@ -217,7 +219,9 @@ const MemoChart = memo(function MemoChart({
   }, [groupKeys, displayType, lineNames, graphResults]);
 
   const sizeRef = useRef<[number, number]>([0, 0]);
-  const timeFormat: TimeFormat = useUserPreferences().timeFormat;
+  const {
+    userPreferences: { timeFormat },
+  } = useUserPreferences();
```

Comment:
I suspect it's out of scope for this PR, but we'll want to update all our charts to support the UTC preference as well, otherwise it'll be confusing that you toggle on UTC mode but the charts will stay in local time. (HDX-681)

I think for now we should just add some copy that it only applies to the search page in the preference modal?